build_extra/rust/rust.gni

```diff
@@ -102,6 +103,10 @@ template("run_rustc") {
       }
     }
 
+    if (defined(invoker.args)) {
+      args += invoker.args
+    }
```

Comment:
I'd prefer not to add this... but it's fine if you can't fix the warnings in `percent_encoding`